### PR TITLE
DCGM receiver: Soft disable the receiver when DCGM not installed

### PR DESCRIPTION
**Pull request by LujieDuan**
b/279022165

Modify the start up behaviour of the DCGM receiver to have similar logic as the NVML receiver - when the receiver is enabled and the DCGM app is not installed OR not running, soft disable the receiver, emit no metrics, and log the warning message. Throwing errors in this case could cause the entire collector to exit, which is normally not the expected behaviour. 

Tests in the Ops Agent: https://github.com/GoogleCloudPlatform/ops-agent/pull/1338

### CODE REVIEW

**Comment by LujieDuan** [comment]
cc @braydonk

**Review by suffiank — APPROVED** [review]
Lgtm :)

**Comment by LujieDuan** [comment]
Hey @braydonk I need a approval with write access :) thanks!

**Review by braydonk — APPROVED** [review]
Did not realize I had to give this too. LGTM!

**Pull request merged**
